### PR TITLE
18MEX: Add mail feature

### PR DESCRIPTION
**Pull request by perwestling**
If a major corporation has trains at start of its OR, add revenue
for the home city to the treasury.

### CODE REVIEW

**Comment by perwestling** [comment]
I considered adding a mail ability, to all corporations, but thought this mail might be too special.

Anyway, the house keeping method addition in the operating is something that I think can be used in more games. E.g. L trains bought by minors in 1822. The term "house keeping" is from 1822.
It is things done at the very start of the OR, before the first tile lay. So in 18MEX it is mail.

File: lib/engine/round/g_18_mex/operating.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative '../operating'
+
+module Engine
+  module Round
+    module G18MEX
+      class Operating < Operating
+        def house_keeping
```
Comment (tobymao) [inline lib/engine/round/g_18_mex/operating.rb:9]: can't you just override start operating? i don't know if house keeping adds that much

Comment (perwestling) [reply lib/engine/round/g_18_mex/operating.rb:9]: That I tried at first. If I do "house_keeping" + super, this means that the "XXX operating YYY" message comes after the mail message.
So I could solve that by overriding skip_steps to do mail + super.
Then I got the idea about 1822, and its housekeeping.

But sure, I can go for the override of skip_steps.

**Comment by perwestling** [comment]
As only 18MEX code was changed after acting upon the review comment, I rebased it as well.

**Review by tobymao — APPROVED** [review]

**Pull request merged**